NIPTool/models/database/sample.py

```diff
@@ -79,13 +79,13 @@ class Sample(BaseModel):
     include: Optional[bool]
     change_include_date: Optional[str] = ""
     comment: Optional[str] = ""
-    status_13: Optional[str]
-    status_18: Optional[str]
-    status_21: Optional[str]
-    status_X0: Optional[str]
-    status_XXX: Optional[str]
-    status_XXY: Optional[str]
-    status_XYY: Optional[str]
+    status_13: Optional[str] = "Normal"
```

Comment:
If statuses are known we can use

```suggestion
    status_13: Literal["Normal", "Abnormal"] = "Normal"
```

Reply:
Add 

`from typing_extensions import Literal`

Reply:
👍